src/ui/public/stringify/types/url.js

```diff
@@ -93,7 +93,7 @@ export function stringifyUrl(Private) {
 
       switch (this.param('type')) {
         case 'img':
-          return '<img src="' + url + '" alt="' + label + '" title="' + label + '">';
+          return `<img src="${url}" alt="A dynamically-specified image located at ${label}">`;
```

Comment:
The original title was just the URL, which doesn't describe the image contents at all. Unfortunately, because this image is dynamically defined and AFAIK we don't support description metadata, the best we can do is to just tell screen readers about where this image came from.